cms/utils/conf.py

```diff
@@ -254,6 +255,12 @@ def get_unihandecode_host():
         return host + '/'
 
 
+def get_placeholder_config():
+    name = 'PLACEHOLDER_CONF'
+    placeholder_conf = getattr(settings, 'CMS_%s' % name, DEFAULTS[name])
+    return OrderedDict(placeholder_conf)
```

Comment:
Dictionaries are not ordered anyway.
And in the current implementation the order does not matter at all.
But casting to an OrderedDict makes easier to implement things in the future without having to change the data structure.